app/templates/administrative-units/new.hbs

```diff
@@ -163,7 +163,8 @@
               <Item @labelFor="purpose">
                 <:label>Doel</:label>
                 <:content>
-                  <RichTextEditor
+                  <AuTextarea
```

Comment:
I guess it's an error due to the merge conflict, but we do want to use the `RichTextEditor` component here